packages/react-table/src/components/TableComposable/examples/ComposableTable.md

```diff
@@ -2010,3 +2011,709 @@ class DraggableTable extends React.Component {
+      'Test cell 6-8',
+      'Test cell 6-9'
+    ],
+    [
+      'Fact 7',
+      'State 7',
+      'Test cell 7-3',
+      'Test cell 7-4',
+      'Test cell 7-5',
+      'Test cell 7-6',
+      'Test cell 7-7',
+      'Test cell 7-8',
+      'Test cell 7-9'
+    ],
+    [
+      'Fact 8',
+      'State 8',
+      'Test cell 8-3',
+      'Test cell 8-4',
+      'Test cell 8-5',
+      'Test cell 8-6',
+      'Test cell 8-7',
+      'Test cell 8-8',
+      'Test cell 8-9'
+    ],
+    [
+      'Fact 9',
+      'State 9',
+      'Test cell 9-3',
+      'Test cell 9-4',
+      'Test cell 9-5',
+      'Test cell 9-6',
+      'Test cell 9-7',
+      'Test cell 9-8',
+      'Test cell 9-9'
+    ]
+  ]);
+  // index of the currently active column
+  const [activeSortIndex, setActiveSortIndex] = React.useState(-1);
+  // sort direction of the currently active column
+  const [activeSortDirection, setActiveSortDirection] = React.useState('none');
+  const onSort = (event, index, direction) => {
+    setActiveSortIndex(index);
+    setActiveSortDirection(direction);
+    // sorts the rows
+    const updatedRows = rows.sort((a, b) => {
+      if (typeof a[index] === 'number') {
+        // numeric sort
+        if (direction === 'asc') {
+          return a[index] - b[index];
+        }
+        return b[index] - a[index];
+      } else {
+        // string sort
+        if (direction === 'asc') {
+          return a[index].localeCompare(b[index]);
+        }
+        return b[index].localeCompare(a[index]);
+      }
+    });
+    setRows(updatedRows);
+  };
+  return (
+    <React.Fragment>
+      <div style={{ height: '600px' }}>
+        <OuterScrollContainer>
+          <InnerScrollContainer>
+            <TableComposable aria-label="Sticky column table" gridBreakPoint="" isStickyHeader>
+              <Thead>
+                <Tr>
+                  {columns.map((column, columnIndex) => {
+                    const sortParams =
+                      columnIndex === 0 || columnIndex === 1
+                        ? {
+                            sort: {
+                              sortBy: {
+                                index: activeSortIndex,
+                                direction: activeSortDirection
+                              },
+                              onSort,
+                              columnIndex
+                            }
+                          }
+                        : {};
+
+                    if (columnIndex === 0) {
+                      return (
+                        <Th key={columnIndex} isStickyColumn modifier="truncate" {...sortParams}>
+                          {column}
+                        </Th>
+                      );
+                    } else if (columnIndex === 1) {
+                      return (
+                        <Th
+                          key={columnIndex}
+                          isStickyColumn
+                          stickyMinWidth="80px"
```

Comment:
I'm noticing a gap between the first and second sticky columns in the last new example. 
Could you also add just a little more detail in the prop descriptions for the sticky minwidth and sticky left offset props? I see the explanation in the examples and it's super helpful, but even just one additional sentence to explain the props' intended uses would be helpful in the api descriptions would help a lot i think.

Reply:
Oops missed updating the width. Will also add the explanation to the prop description.